docs/src/test/java/jdbi/doc/KotlinPluginTest.kt

```diff
@@ -27,7 +30,10 @@ import kotlin.test.assertEquals
 
 class KotlinPluginTest {
     @Rule @JvmField
-    val db = H2DatabaseRule().withPlugins()
+    val db = H2DatabaseRule()
+        .withPlugin(SqlObjectPlugin())
+        .withPlugin(KotlinPlugin())
+        .withPlugin(KotlinSqlObjectPlugin())
```

Comment:
`withPlugins()` was picking up postgres plugin, which tried to use Connection.unwrap(), which H2 connection doesn't support